.golangci.yml

```diff
@@ -0,0 +1,60 @@
+# This file configures github.com/golangci/golangci-lint.
+
+run:
+  timeout: 3m
+  tests: true
+  # default is true. Enables skipping of directories:
+  #   vendor$, third_party$, testdata$, examples$, Godeps$, builtin$
+  skip-dirs-use-default: true
+
+service:
+  golangci-lint-version: 1.49.0
+
+linters:
+  disable-all: true
+  enable:
+    - whitespace # Tool for detection of leading and trailing whitespace
+    - wsl # Whitespace Linter - Forces you to use empty lines!
```

Comment:
No need to use exclamation marks for comments

Reply:
Removed